### PR TITLE
Registry v2.8.0 release

### PR DESCRIPTION
**Pull request by milosgajdos**
### Release notes
https://github.com/distribution/distribution/releases/tag/v2.8.0

### docker-library docs
https://github.com/docker-library/docs/pull/2113

Signed-off-by: Milos Gajdos <milosthegajdos@gmail.com>

### CODE REVIEW

**Review by thaJeztah — COMMENTED** [review]
SGTM

File: library/registry [hunk 1]
```diff
@@ -2,15 +2,12 @@ Maintainers: Milos Gajdos (@milosgajdos),
              Sebastiaan van Stijn (@thajeztah)
 GitRepo: https://github.com/docker/distribution-library-image.git
 
+Tags: 2.8.0, 2.8, 2, latest
+Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
+GitFetch: refs/heads/master
+GitCommit: 5fd2614beccba35242694b13f8a37653c05ea852
+
 Tags: 2.8.0-beta.1
```
Comment (tianon) [inline library/registry:10]: Any particular reason to keep this version listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Comment (milosgajdos) [reply library/registry:10]: No particular reason, we just followed the existing setup -- that being said, `beta` release is _identical_ from th stable one. I'm happy to remove it. Do you want me to?

Comment (milosgajdos) [reply library/registry]: Removed https://github.com/docker-library/official-images/pull/11821/commits/dfb8057d207f261bf6011fb5fe98988b80799986

**Comment by tianon** [comment]
Here's a better diff (slightly adjusted `git diff` parameters):

<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-cat b/_bashbrew-cat
index 2082b01..7396cd1 100644
--- a/_bashbrew-cat
+++ b/_bashbrew-cat
@@ -1,14 +1,6 @@
 Maintainers: Milos Gajdos (@milosgajdos), Sebastiaan van Stijn (@thajeztah)
 GitRepo: https://github.com/docker/distribution-library-image.git
 
-Tags: 2.7.1, 2.7, 2, latest
-Architectures: amd64, arm32v6, arm64v8
-GitFetch: refs/heads/release/2.7
-GitCommit: ab00e8dae12d4515ed259015eab771ec92e92dd4
-amd64-Directory: amd64
-arm32v6-Directory: arm
-arm64v8-Directory: arm64
-
-Tags: 2.8.0-beta.1
+Tags: 2.8.0, 2.8, 2, latest
 Architectures: amd64, arm32v6, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: c70c2acb365928f10bfc7706cc0f3f4d16f7427b
+GitCommit: 5fd2614beccba35242694b13f8a37653c05ea852
diff --git a/_bashbrew-list b/_bashbrew-list
index 7a40cf7..206c405 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,5 +1,4 @@
 registry:2
-registry:2.7
-registry:2.7.1
-registry:2.8.0-beta.1
+registry:2.8
+registry:2.8.0
 registry:latest
diff --git a/registry_2.8.0-beta.1/config-example.yml b/registry_2.8.0-beta.1/config-example.yml
deleted file mode 100644
index 3277f9a..0000000
diff --git a/registry_2.8.0-beta.1/docker-entrypoint.sh b/registry_2.8.0-beta.1/docker-entrypoint.sh
deleted file mode 100755
index 319d1fd..0000000
diff --git a/registry_2.8.0-beta.1/Dockerfile b/registry_latest/Dockerfile
similarity index 54%
rename from registry_2.8.0-beta.1/Dockerfile
rename to registry_latest/Dockerfile
index c7dfb35..d73c9b2 100644
--- a/registry_2.8.0-beta.1/Dockerfile
+++ b/registry_latest/Dockerfile
@@ -4,15 +4,15 @@ RUN apk add --no-cache ca-certificates
 
 RUN set -eux; \
 # https://github.com/distribution/distribution/releases
-	version='2.8.0-beta.1'; \
+	version='2.8.0'; \
 	apkArch="$(apk --print-arch)"; \
 	case "$apkArch" in \
-		x86_64)  arch='amd64';   sha256='d69d68406a30466070d8e47c21fafaadae5c048239df41b50b28a79fa12945ab' ;; \
-		aarch64) arch='arm64';   sha256='2a080694ae2528a628245cf5177d4d2e55d430c5dd76cbbf61e5aee08e75abf6' ;; \
-		armhf)   arch='armv6';   sha256='b5092556fd196f59c7d2a8d4e4460c193588d9f7f39d825d17f299d7fc856ca1' ;; \
-		armv7)   arch='armv7';   sha256='f479d7d42a4c6086ee9a51f605386fcfb953198bf9ab48c515ffdde33ad46e5d' ;; \
-		ppc64le) arch='ppc64le'; sha256='dc0444e672511b4f8dc19fcec1624a33fdaf49a72ff3c1a69bae9bd3399cd074' ;; \
-		s390x)   arch='s390x';   sha256='4b814d1cb60ee7881e59c1ee52635755c2279196861892cface92e58aa6ac749' ;; \
+		x86_64)  arch='amd64';   sha256='7b2ebc3d67e21987b741137dc230d0f038b362ba21e02f226150ff5577f92556' ;; \
+		aarch64) arch='arm64';   sha256='16b9f497751bd3abe8b75d0f1538e2767ef3c536f4f11d05a312fb1767d43e85' ;; \
+		armhf)   arch='armv6';   sha256='5021831ba045a3cc409f6f62ab50c04db2c935a058eb53ce9d74a4dd5ba41102' ;; \
+		armv7)   arch='armv7';   sha256='ff659c577266662edb247d4719399fa1179bfcb90fb6006fc63396b7089c0f70' ;; \
+		ppc64le) arch='ppc64le'; sha256='46fbd645b415c68222ee0e8043a91c27b6bb2ec2e0a568f663d1e78cc69d8cda' ;; \
+		s390x)   arch='s390x';   sha256='ebbd08228cf290ceef50ab542ae6087b66173b18fa84868210cbbdb458d11bd3' ;; \
 		*) echo >&2 "error: unsupported architecture: $apkArch"; exit 1 ;; \
 	esac; \
 	wget -O registry.tar.gz "https://github.com/distribution/distribution/releases/download/v${version}/registry_${version}_linux_${arch}.tar.gz"; \
diff --git a/registry_latest/registry b/registry_latest/registry
deleted file mode 100755
index 8611d0a..0000000
```

</details>

**Comment by milosgajdos** [comment]
@tianon can we get https://github.com/docker-library/docs/pull/2113 merged in too? It mostly just updates some links to some URLs :)

**Pull request merged**